Datadog.Trace.sln

```diff
@@ -285,10 +285,16 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Samples.AspNetCoreMvc31", "
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Samples.AspNetCoreMvc21", "samples\Samples.AspNetCoreMvc21\Samples.AspNetCoreMvc21.csproj", "{D141BD06-DD95-4CAF-85CD-657116E0DAD4}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GenerateAzureSiteExtension", "tools\GenerateAzureSiteExtension\GenerateAzureSiteExtension.csproj", "{454B9EA7-9B9A-4CB8-B019-988579533BC1}"
```

Comment:
What's the reasoning for keeping the GenerateAzureSiteExtension in this repo? I thought this was being separated out into a new repo

Reply:
We still need to generate the binaries/artifacts for download.

Reply:
It might be good to rename, as from our perspective, all it does is drop a tracer home directory somewhere.